lib/security/aes.js

```diff
@@ -0,0 +1,42 @@
+var CryptoJS = require("crypto-js");
+var q = require('q');
+
+/**
+* Encrypt an Object using AES with a specified secret key.
+* @param data - the data to encrypt.
+* @param key - the secret key to encrypt with.
+*/
+function encrypt(data, key) {
```

Comment:
Is encryption / decryption an asynchronous operation?

If not, the promises are not required